examples/default/s3/example.tf

```diff
@@ -0,0 +1,44 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "lambda" {
+  source = "../../../"
+
+  name_prefix = "example"
+  s3_bucket   = "s3-bucket-name"
+  s3_key      = "path-to-function"
```

Comment:
Should we create a `hello_world` function in the OSS bucket to use in this example?

Reply:
That is a great idea! But preferably, all artifacts in that bucket should be built via e.g. Travis, even though it's just a simple python/node.js "hello world"?

Reply:
Agree, will create the function and build/deploy vi Travis.